src/main/java/nl/thehyve/whereabouts/controllers/InstanceController.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019 The Hyve
+ * This file is distributed under the MIT License (see accompanying file LICENSE).
+ */
+
+package nl.thehyve.whereabouts.controllers;
+
+import nl.thehyve.whereabouts.domains.Instance;
+import nl.thehyve.whereabouts.exceptions.InstanceNotFoundException;
+import nl.thehyve.whereabouts.repositories.InstanceRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@RestController
+public class InstanceController {
+
+    private final InstanceRepository instanceRepository;
+
+    InstanceController(InstanceRepository instanceRepository) {
+        this.instanceRepository = instanceRepository;
+    }
+
+    @GetMapping("/instances")
+    List<Instance> getInstances() {
```

Comment:
If you make the controller endpoints `public`, IntelliJ won't complain about unused methods.